pkg/action/build-uki.go

```diff
@@ -630,13 +655,15 @@ func (b *BuildUKIAction) imageFiles(sourceDir string) (map[string][]string, erro
 	// the keys are the target dirs
 	// the values are the source files that should be copied into the target dir
 	data := map[string][]string{
-		"EFI":            {},
-		"EFI/BOOT":       {filepath.Join(sourceDir, "BOOTX64.EFI")},
-		"EFI/kairos":     {},
-		"EFI/tools":      {},
-		"loader":         {filepath.Join(sourceDir, "loader.conf")},
-		"loader/entries": {},
-		"loader/keys":    {},
+		"EFI":                 {},
+		"EFI/BOOT":            {filepath.Join(sourceDir, "BOOTX64.EFI")},
+		"EFI/systemd":         {},
+		"EFI/systemd/drivers": {filepath.Join(sourceDir, "ext2_x64.efi")},
```

Comment:
what about the arm one?

Reply:
no arm support for now until we can build it ourselves. Also the BOOTX64.efi file points to no arm support as well, lol